packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -227,19 +223,25 @@ class TimeInput extends React.Component {
   handleSecondFocus(event) {
     this.handleFocus(event);
     this.setState({ secondInitialFocused: true });
-    this.secondInput.setSelectionRange(0, this.secondInput.value.length);
+    if (!TimeUtil.isConsideredMobileDevice()) {
```

Comment:
Whats the reason for wrapping these in the condition check?

Reply:
The `setSelectionRange` call sans this check brings up the contextual menu (see below) on iOS every time the focus shifts.

![context-menu-ios](https://user-images.githubusercontent.com/10090859/64232566-ce0e1800-cf0f-11e9-9032-b49d20ed338c.png)

Reply:
Ah, let's add a code comment about this before the conditional so we don't accidentally remove it without understanding its intended use.